core/script_debugger_remote.cpp

```diff
@@ -67,17 +67,20 @@ Error ScriptDebuggerRemote::connect_to_host(const String &p_host, uint16_t p_por
 
 	int port = p_port;
 
-	int tries = 3;
+	int tries = 6;
+	int waits[6] = { 1, 10, 100, 1000, 1000, 1000 };
```

Comment:
Maybe best to use `waits[tries]` here so that if someone changes only `tries` to e.g. 5, we get a compile error instead of a crash?

Reply:
Good point. Changed.